docs/Using-EthSigner/Azure.md

```diff
@@ -30,6 +30,10 @@ option set to `8590` to avoid conflict with the default EthSigner listening port
     besu --network=dev --miner-enabled --miner-coinbase=0xfe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --host-whitelist=* --rpc-http-enabled --rpc-http-port=8590 --data-path=/tmp/tmpDatdir
     ```
 
+!!! important 
+    To use EthSigner for signing transactions, the Ethereum client genesis file must include at least the 
```

Comment:
My version?
EthSigner _requires_ a chainId to be used when signing transactions, as such, it is only capable of submitting transactions to a blockchain which is in (or beyond) the Byzantium milestone (i.e. chainId is used for replay prevention). [Is it byzantium?!]

Reply:
"which has passed the byzantium milestone" maybe?

Reply:
Think its actually the SpuriousDragon milestone (not Byzantium <-- is way too new)